domain/domain_windows_test.go

```diff
@@ -0,0 +1,160 @@
+//go:build windows
```

Comment:
These tests only run on Windows since `filepath` behaviour is platform dependent.

Reply:
@dchw do we have a way to schedule tests like this to run on our Windows VM, or something like that?

Reply:
No; we would need to configure an additional Windows runner.

Reply:
Alright, perhaps a goal for us another day